### PR TITLE
blog: offline snapshot introduction post

### PR DESCRIPTION
**Pull request by milas**
Offline snapshots + Tilt Cloud deprecation

### CODE REVIEW

File: blog/_posts/2022-05-12-offline-snapshots.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+image: "/assets/images/offline-snapshots/title.jpg"
+image_caption: 'Photo by <a rel="noopener noreferrer" href="https://unsplash.com/photos/jvPTh8YAgYM">
+TheRegisti</a> on Unsplash'
+description: "Same Snapshots, New Workflow"
+tags:
+- tilt
+- snapshot
+---
+
+Starting with Tilt v0.30.0, Snapshots will work a little differently!
```
Comment (hyu) [inline blog/_posts/2022-05-12-offline-snapshots.md:16]: Maybe link to the Releases page?

**Review by hyu — APPROVED** [review]
Looks good!

File: blog/_posts/2022-05-12-offline-snapshots.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+tilt snapshot view snapshot.json
+```
+
+After a moment, you should see your Snapshot in the browser:
+![Screenshot of a Tilt Snapshot viewed locally](/assets/images/offline-snapshots/snapshot.png)
+
+
+### Tilt Cloud Shutdown
+Starting May 19, 2022, we'll move Tilt Cloud into read-only mode.
+No new users will be able to register, no new teams can be created, and most importantly, no new cloud Snapshots can be saved.
```
Comment (lizzthabet) [inline blog/_posts/2022-05-12-offline-snapshots.md:52]: is it worth mentioning anything about the `set_team` built-in deprecation? (just thinking if there are other questions users might have!)

**Review by lizzthabet — APPROVED** [review]
this looks great!

**Pull request merged**